meta-balena-common/recipes-devtools/go/go-cross.inc

```diff
@@ -1,9 +1,9 @@
 inherit cross
 
-PROVIDES = "virtual/${TARGET_PREFIX}go"
+PROVIDES = "virtual/${TUNE_PKGARCH}-go"
 DEPENDS = "go-native"
```

Comment:
This change I would like to understand. go-cross is included in recipes that run in the host but output code for the target. I think using `TARGET_PREFIX` is correct, and `TUNE_PKGARCH` is strange. I am wondering if now that this is working you could revert this change see if it is still needed.

Reply:
So, this is explained in https://patchwork.openembedded.org/patch/156028/